index.html

```diff
@@ -11,24 +11,33 @@
     <title>Document</title>
 </head> 
 <body> 
-    <header>
-        <ul>
-            <li><img src="images/Logo.png" alt="logo"></li>
-            <li><i class="fa-solid fa-bars"></i></li>
-        </ul>
-    </header>
-    <section class="Headline"> 
-        <h1>I’m kelvin Glad to see you!</h1>
-        <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <p id="two">LET'S CONNECT</p>
-        <a href="#"></a>
-        <ul> 
-            <li><a class="fa-brands fa-twitter" href="https://twitter.com/kevin_ngotho?s=09"></a></li>
-            <li><a class="fa-brands fa-linkedin" href="https://www.linkedin.com/in/kelvin-kaviku-5178001a6"></a></li>
-            <li><a class="fa-brands fa-medium" href="#"></a></li>
-            <li><a class="fa-brands fa-github" href="https://github.com/brainskev"></a></li>
-            <li><a class="fa-solid fa-hand-peace" href="#"></a></li>
-        </ul>
-    </section>
+    <div class="flex-container">
+        <div style="align-self: left"></div>
+            <header>
+                <ul>
+                    <li><img src="images/menu.png" alt="logo"></li>
+                    <!--<li><i class="fa-solid fa-bars"></i></li>-->
+                </ul>
+            </header>
+        </div>
+        <div>
+            <section class="Headline"> 
+                <h1>I’m kelvin Glad to see you!</h1>
+                <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p id="two">LET'S CONNECT</p>
+                <a href="#"></a>
```

Comment:
- [ ] Good job adding a background image but you have set it to the `body` instead of this `Headline` section. Kindly remove it from the body and set it to this section accordingly.
![image](https://user-images.githubusercontent.com/29071984/198317666-422b44c5-f9dc-4300-9d07-76fc0008da41.png)